pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -163,15 +157,42 @@ let
   # Python packages built through cross-compilation are always for the host platform.
   disallowedReferences = lib.optionals (python.stdenv.hostPlatform != python.stdenv.buildPlatform) [ python.pythonForBuild ];
 
+  passthru = passthru // {
+    tests.installation = stdenv.mkDerivation {
```

Comment:
maybe better to include the whole `self` here `stdenv.mkDerivatoin (self // {... })`